local/code/modules/surgery/mechanic_steps.dm

```diff
@@ -0,0 +1,161 @@
+//cut wires
+/datum/surgery_step/cut_wires
+	name = "cut wires"
+	implements = list(
+		TOOL_WIRECUTTER = 100,
+		TOOL_SCALPEL = 75,
+		/obj/item/knife	= 50,
+		/obj/item = 10,
+	) // 10% success with any sharp item.
+	time = 2.4 SECONDS
+	preop_sound = 'sound/items/tools/wirecutter.ogg'
+
+/datum/surgery_step/cut_wires/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	display_results(
+		user,
+		target,
+		span_notice("You begin to cut loose wires in [target]'s [parse_zone(target_zone)]..."),
+		span_notice("[user] begins to cut loose wires in [target]'s [parse_zone(target_zone)]."),
+		span_notice("[user] begins to cut loose wires in [target]'s [parse_zone(target_zone)]."),
+	)
+
+/datum/surgery_step/cut_wires/tool_check(mob/user, obj/item/tool)
+	if(implement_type == /obj/item && !tool.get_sharpness())
```

Comment:
this check is never going to succeed because its literally only checking for `/obj/item` and not subtypes, but if you change it to `istype` its going to fail for wirecutters because they don't have a defined sharpness, this probably just needs to be rethought